reference/features/configuration.md

```diff
@@ -243,7 +250,7 @@ getconf PAGE_SIZE
 **CLI option**: `--http-payload-size-limit`
 **Default value**: `104857600` (~100MB)
 
-Sets the maximum size of accepted JSON payloads. Value must be given in bytes.
+Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit.
```

Comment:
```suggestion
Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit.
```
Maybe also add here an example of the default value written in different units, as in the max-index-size and max-udb-size sections

Reply:
```suggestion
Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
```